packages/pluggableWidgets/slider-web/src/utils/useOnChangeDebounced.ts

```diff
@@ -0,0 +1,21 @@
+import { Big } from "big.js";
+import { useCallback, useMemo, useRef } from "react";
+import { debounce, executeAction } from "@mendix/piw-utils-internal";
+import { SliderContainerProps } from "../../typings/SliderProps";
+
+export function useOnChangeDebounced(props: SliderContainerProps) {
+    const valueObjRef = useRef(props.valueAttribute);
+    valueObjRef.current = props.valueAttribute;
+
+    const onChangeEnd = useMemo(() => debounce(() => executeAction(props.onChange), 400), [valueObjRef]);
```

Comment:
Maybe 500?

Reply:
Don't know tbh, this is just standard value that I use, if you think 500 is better, then I can change